distribution/lib/Standard/Base/0.0.0-dev/src/Data/XML.enso

```diff
@@ -311,8 +311,7 @@ type XML_Element
         XML_Error.handle_java_exceptions <|
             XML_Utils.innerXML self.java_element
 
-    ## GROUP Metadata
-       Gets elements matching a given tag name.
+    ## Gets elements matching a given tag name.
```

Comment:
What are the +- guidelines for when a method gets assigned to a group VS it has no group?